checker/cmd/aelpackage/keygen.go

```
@@ -0,0 +1,134 @@
+	} else if info.Mode()&os.ModeSymlink != 0 || !info.IsDir() {
+		return keygenResult{}, fmt.Errorf("%s trust directory must be a real directory", role)
+	}
+
+	privateText := []byte(base64.StdEncoding.EncodeToString(private) + "\n")
+	if err := writeKeyFileExclusive(privatePath, privateText, 0o600); err != nil {
+		return keygenResult{}, fmt.Errorf("write private key: %w", err)
+	}
+	publicText := []byte(base64.StdEncoding.EncodeToString(public) + "\n")
+	if err := writeKeyFileExclusive(publicPath, publicText, 0o644); err != nil {
+		if cleanupErr := os.Remove(privatePath); cleanupErr != nil {
+			return keygenResult{}, fmt.Errorf("write public key: %v; remove incomplete private key: %w", err, cleanupErr)
```

golangci-lint (2.12.2)

[error] non-wrapping format verb for fmt.Errorf. Use `%w` to format errors

(errorlint)